app/component/summary-row.scss

```diff
@@ -90,8 +90,9 @@
     flex-grow: 1; //Needed on IE
 
     .leg {
-      flex: 1 0 0;
```

Comment:
Seemed to me that the page worked fine even without the fix so I have removed it for now.